roles/deprovision-cordova-app/tasks/main.yml

```diff
@@ -1,3 +1,11 @@
 # Delete cordova app representation using the mobile cli
+- set_fact: app_not_found_err="'mobileclients.mobile.k8s.io \"{{ appName }}-cordova\" not found'"
```

Comment:
This can be `set_fact: app_not_found_err='mobileclients.mobile.k8s.io "{{ appName }}-cordova" not found'`